Blinkt.js

```diff
@@ -12,13 +12,23 @@ Blinkt = function () {};
  * before any other commands. All pixels will start off white at
  * full brightness by default.
  */
-Blinkt.prototype.setup = function setup () {
+Blinkt.prototype.setup = function setup (dat, clk) {
 	// Set WPI to GPIO mode
 	wpi.setup('gpio');
 
+	
+	if(!!dat && isNaN(dat)){
```

Comment:
Do we need to do a !!dat here? I would have thought that if (dat && isNaN(dat)) would be ok? If casting to a boolean is required for some reason then ESLint will complain about !!, better to explicit cast to boolean e.g. 

`if (Boolean(dat) && isNaN(dat)) { ... }`

Reply:
That's fine, I'll switch to a Boolean cast. I usually use !! just because it uses up less screen space, but Boolean() is definitely more readable.